doc/source/news.rst

```diff
@@ -7,6 +7,79 @@
 News
 ====
 
+.. _release-8-0-2:
+
+Release 8.0.2 - 2018-04-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/executables/grndb`][:ref:`grndb-force-truncate`] Improved
+  ``grndb recover --force-truncate`` option that it can be truncated even if
+  locks are left on the table.
+
+* [Windows] Use ``CONDITION_VARIABLE`` instead of Groonga's original implementation.
+
+* [:doc:`/reference/commands/logical_range_filter`] Added ``sort_keys`` option.
+
+* Added a new function ``time_format()``.
+  You can specify time format against a column of Time type.
+  You can specify with use format of ``strftime`` .
+
```

Comment:
Add a link to `time_format()` documentation is good if you afford to do.
`Time` is better to markup as <pre>\`Time\`</pre>